HW4/services/__tests__/public-holidays.service.test.ts

```diff
@@ -0,0 +1,165 @@
+import axios from "axios";
+
+import {
+  getListOfPublicHolidays,
+  checkIfTodayIsPublicHoliday,
+  getNextPublicHolidays,
+} from "../public-holidays.service";
+import { PUBLIC_HOLIDAYS_API_URL } from "../../config";
+import {
+  YEAR,
+  INCORRECT_YEAR,
+  COUNTRY,
+  INCORRECT_COUNTRY,
+  PUBLIC_HOLIDAYS_DATA,
+  PUBLIC_HOLIDAYS_SHORTEN_DATA,
+} from "../../mocks";
+
+describe("public-holidays.service: ", () => {
+  describe("getListOfPublicHolidays: get list of public holidays by year and country", () => {
+    test("should return public holidays", async () => {
+      // mock response from API
+      jest
```

Comment:
It's better to use https://www.npmjs.com/package/axios-mock-adapter for axios mocking